tests/test_cpu_load.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import logging
+import time
+
+from lcrs_embedded.models import ScanResult
+from . import *  # noqa  # This loads the fixtures that are in __init__.py
+
+logger = logging.getLogger(__name__)
+
+
+def test_cpu_load(runserver):
+    """
+    Test duration of load of CPU.
+    """
+    from lcrs_embedded.system.cpu_load import cpu_load
+    logger.info("Testing duration of load of CPU with two cores")
+    duration_in_seconds = 1
+    startTime = time.time()
+    scan_result = ScanResult(
+        processor_cores=2
+    )
+    cpu_load(scan_result, duration_in_seconds)
+    actual_duration = time.time() - startTime
+    assert actual_duration >= duration_in_seconds
+    assert actual_duration <= duration_in_seconds + 1
```

Comment:
Cool! I like this test, it really hits the nail on what's expected to happen in a nice way and with out being unnecessarily slow!